src/en/proposals/slambamactionman-tourist.md

```diff
@@ -0,0 +1,63 @@
+
+- **Visit:** The Tourist must have visited certain locations on the station. This could be certain departments (Security, Engineering) or specific sub-areas in low-security deparments (Morgue, Cargo Bay, Robotics). Areas should not be high-security areas; Vault, Armory or Head Offices are examples of these.
+- **Get Drunk in Bar:** The Tourist must have gotten drunk in the bar. Like Visit but with an extra objective.
+- **Photograph Crew:** The Tourist must have used their Camera item on a randomly selected crewmember. Camera described further below.
+- **Photograph Item:** The Tourist must have used their Camera near a specific item; the item *can* be held by a person or be worn, but being inside a container does not count. The item may be restricted, but should not require major larceny to photograph if the owner is unavailable or unwilling. (e.g. Node Analyzer/Seclite vs. Hand Teleporter/R&D Server)
+- **Stamp Collecting:** The Tourist must end the round with a paper in their inventory that has X different job stamps on it. Generic stamps such as Approved/Denied do not count, but non-Command jobs such as Chaplain, Clown or Syndicate do.
+- **Eat:** The Tourist must have eaten a specific kind of dish. This dish should not rely on too exotic/unreliable ingredients (Carps/Spiders/Xeno/Amanita) but still require effort from the Chef (certain burgers, pies, pancakes and cakes). Only available if a Chef is on the station.
+- **Pet:** The Tourist must have given a specific station pet a pet. Functionally quite similar to Visit, but less static.
+
+The Tourist is *not* able to spawn as an antag. That instead falls onto existing antags, who may choose to fake being a Tourist for the benefits it provides. A Tourist may become an antag during the course of the round via external factors (e.g. via a conversion antag).
+
+### The Travel Camera / Polaroid Camera
+
+The Travel Camera is an item that the Tourist spawns with and is semi-exclusive to the role. Its main functionality is to help fulfill the Photograph objective, however it can also be used as a minor defensive tool should the Tourist be at risk of harm. 
+
+The Travel Camera acts very similar to a Flash with reduced stun duration, and contains 10 charges. The Travel Camera can not be refilled (you only brought a single roll!), so Tourists are discouraged from spamming it and to instead use it to fulfill their objective, with any leftover uses being free for roleplay/defense.
+
+The Travel Camera is also rare maintenance loot and available in the Syndicate Uplink, to ensure that it can not be used as a way to verify whether someone is a normal Tourist or not. If proper photography functionality is implemented the Camera may find use in other jobs as well. The uplink Camera is the same as a maintenance/Tourist camera. The Camera can NOT be used by Head Revs to convert crew.
+
+The Travel Camera is also a cosmetic item, in that it can be equipped in the tie slot to hang around the Tourist's neck.
```

Comment:
This is a little sus BUT it should be fine if the delay is long enough and the duration of the effect is short enough.

Reply:
The Travel Camera is a roleplay/objective tool first and weapon second. Its flash capabilities should be underwhelming to begin wtih and if it ends up being too oppressive it can be further nerfed.